packages/react-dom/src/__tests__/ReactNestedUpdateBug-test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ * @jest-environment node
+ */
+
+// This is a regression test for https://github.com/facebook/react/issues/13188.
```

Comment:
Nit: what do you think about dropping the period after this link so that it's easier to copy/paste from Github?